x/group/errors.go

```diff
@@ -0,0 +1,16 @@
+package group
+
+import (
+	"github.com/cosmos/cosmos-sdk/types/errors"
+)
+
+var (
+	ErrEmpty        = errors.Register(ModuleName, 202, "value is empty")
```

Comment:
why 200's here?

Reply:
That was part of the old code, I don't see any particular reason to keep it as is, replaced with 2, 3, etc.